backend/model/channel/src/main/java/co/airy/model/channel/dto/ChannelContainer.java

```diff
@@ -0,0 +1,33 @@
+package co.airy.model.channel.dto;
+
+import co.airy.avro.communication.Channel;
+import co.airy.model.metadata.dto.MetadataMap;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.util.List;
+
+import static co.airy.model.metadata.MetadataRepository.newChannelMetadata;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChannelContainer implements Serializable {
+    private Channel channel;
+    private MetadataMap metadataMap;
+
+    public MetadataMap getMetadataMap() {
+        return this.metadataMap != null ? this.metadataMap : getDefaultMetadata();
+    }
+
+    private MetadataMap getDefaultMetadata() {
```

Comment:
This ensures that there will always be a `channel.metadata.name` for the clients to use